yt_idefix/fields.py

```diff
@@ -20,11 +23,23 @@ def setup_fluid_fields(self):
         setup_magnetic_field_aliases(
             self, "idefix-vtk", [f"BX{idir}" for idir in "123"]
         )
+        for idir in "123":
+            self.alias(
```

Comment:
why is it needed after `setup_magnetic_field_aliases` ?

Reply:
As far as I can remember, magnetic fields had to be referred to as as for example, ("pluto-vtk", "BX1") and not as conveniently by ("gas", "BX1"). I thought that these lines would perhaps account for that convenience.

Reply:
`("gas", "BX1")` should *not* be allowed. It's important to keep a clear distinction between the `"gas"` field type, which should contain fields whose name do not depend on specific conventions used in different codes, and the other, more flexible "pluto-vtk" field type which is explicitly meant to match field names as they are found on disk. 
Please remove this as well as similar blocks in other classes.